Project.toml

```diff
@@ -1,14 +1,14 @@
 name = "LimitedLDLFactorizations"
 uuid = "f5a24dde-3ab7-510b-b81b-6a72c6098d3b"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
```suggestion
version = "0.3.1"
```

Reply:
We didn't do any release since 2 years and a half Dominique.

Reply:
That's not how releases work.